### DISCUSSION TITLE
Where is Server Data Located?

### DISCUSSION THREAD

**Discussion by JasonTheLucky** [excalidraw/excalidraw#7140] [General]
We'd like to use excalidraw but need to know which country houses the data we create on excalidraw. I couldn't find anything that covers this. I looked at the Privacy Policy of Excalidraw+ but did not see it there either.

Can you help me out?

Thanks,
Jason

**Comment by PseudoResonance** [comment]
The public Excalidraw app stores your drawing information on Google Firebase, wherever that is. I don't know exactly how Firebase works, but I don't think you can narrow down a specific country responsible. It's wherever Google hosts their datacenters and wherever your data happens to be moved to. It is encrypted on your end though, so Google can't exactly see what you put there.
